README.md

```diff
@@ -1 +1,25 @@
-# mysqld_exporter (UNDER CONSTRUCTION)
+# mysqld_exporter
+
+Exporter for MySQL server metrics http://prometheus.io/
+
+## Building and running
+
+    make
+    ./mysqld_exporter <flags>
+
+## Configuration
+The configuration is in JSON. An example with all possible options:
```

Comment:
I think it might be better to be configurable entirely via flags rather than having to mess with json, there's not many possible options.